src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -10502,5 +10502,105 @@ .maxstack  1
     }
 ");
         }
+
+        [Fact]
+        public void Records_AddWellKnownMember()
```

Comment:
These fail. Still trying to work out why. It appears to be a failure of symbol matching, but the symbol matching tests I added for the same scenario passes.

Reply:
@davidwengier I'm completely unsure. But it could be because the parameter of PrintMembers is `sb` instead of `builder`?

Reply:
Argh!! Thank you thank you thank you <3

Spent so long staring at the test code, forgot to look at the test input 🤦‍♂️